.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "deps/pybind11"]
+	path = deps/pybind11
+	url = ../../pybind/pybind11
+	branch = stable
```

Comment:
I think `pybind11` is included in `eic-shell`, so no need to make a submodule. You can find its headers in the container at `/usr/local/include/pybind11/`.